packages/rest/src/coercion/coerce-parameter.ts

```diff
@@ -32,7 +32,14 @@ export function coerceParameter(
   data: string | undefined | object,
   spec: ParameterObject,
 ) {
+  if (!spec.schema && spec.content) {
+    const content = spec.content;
+    const jsonSpec = content['application/json'];
+    spec.schema = jsonSpec.schema;
```

Comment:
This is ok in a spike, but please note that the real implementation must not mutate (modify) the `spec` object given to us by the caller.